Lib/test/test_class.py

```diff
@@ -873,12 +877,31 @@ def __init__(self):
         self.c = 3
         self.d = 4
 
+class VarSizedSubclass(tuple):
+    pass
+
 
 class TestInlineValues(unittest.TestCase):
 
-    def test_flags(self):
```

Comment:
Please revive test_flags also.

Reply:
Ah, I see, Should I drop this commit then?